expressmailing.php

```diff
@@ -98,13 +101,16 @@ public function install($alter_db = true)
 			&& $this->installAdminTab('AdminMarketingSList', false, $this->l('Sms-Mailing'), 'AdminMarketingS')
 			&& $this->installAdminTab('AdminMarketingEStats', false, $this->l('E-Mailing'), 'AdminMarketingE')
 			&& $this->installAdminTab('AdminMarketingFStats', false, $this->l('Fax-Mailing'), 'AdminMarketingF')
-			&& $this->installAdminTab('AdminMarketingSStats', false, $this->l('Sms-Mailing'), 'AdminMarketingS');
+			&& $this->installAdminTab('AdminMarketingSStats', false, $this->l('Sms-Mailing'), 'AdminMarketingS')
+			/* Add QuickLink */
+			&& $this->installQuickAccess();
```

Comment:
I'm sorry but do not add your module in the quick access. The link is already in the menu Price Rules, which is enough. If every module inserted its link in this list, it would loose its meaning.